src/include/global_kvs.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Filename: global_kvs.h
+ * Description: Declaration for NSAL global kvs api
+ *
+ * Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ * For any questions about this software or licensing,
+ * please email opensource@seagate.com or cortx-questions@seagate.com.
+ */
+ 
+/** Global KV-Store operations **/
+/* The functionality provided by below APIs is dependent on basic
+ * framework provided kvstore.h
+ */ 
+
+#ifndef GLOBAL_KVS_H
+#define GLOBAL_KVS_H
+
+
+/* Set an index mentioned by fid_str as global kvstore*/
+int nsal_set_global_kvs(const char *fid_str);
```

Comment:
Question: The caller passing the index fid passed here should be careful that the index must be created first. This function only opens the index and will fail if the caller passes a non-existent index.

Shouldn't we mention it in the description of this function?

Reply:
Sure .. I will add that, good catch.